<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>stackedWidget1</class>
  <widget class="QStackedWidget" name="stackedWidget1">
    <property name="windowModality">
      <enum>Qt::NonModal</enum>
    </property>
    <property name="enabled">
      <bool>true</bool>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>274</width>
        <height>278</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="windowTitle">
      <string>StackedWidget</string>
    </property>
    <property name="currentIndex">
      <number>1</number>
    </property>
    <widget class="QWidget" name="form1">
      <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
        <item>
          <widget class="QLabel" name="label1">
            <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <property name="text">
              <string>label</string>
            </property>
            <property name="alignment">
              <set>Qt::AlignCenter</set>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="button1">
            <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <property name="text">
              <string>button</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPlainTextEdit" name="textview1">
            <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
    <widget class="QWidget" name="dialog1">
      <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
          <spacer name="verticalSpacer">
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>20</width>
                <height>110</height>
              </size>
            </property>
          </spacer>
        </item>
        <item>
          <widget class="QLineEdit" name="entry1">
            <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
              </sizepolicy>
            </property>
            <property name="minimumSize">
              <size>
                <width>120</width>
                <height>28</height>
              </size>
            </property>
            <property name="maximumSize">
              <size>
                <width>16777215</width>
                <height>28</height>
              </size>
            </property>
          </widget>
        </item>
        <item>
          <spacer name="verticalSpacer_2">
            <property name="orientation">
              <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
              <size>
                <width>20</width>
                <height>110</height>
              </size>
            </property>
          </spacer>
        </item>
      </layout>
    </widget>
  </widget>
  <resources />
  <connections>
    <connection>
      <sender>form1</sender>
      <signal>destroyed()</signal>
      <receiver>form1</receiver>
      <slot>close()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>203</x>
          <y>138</y>
        </hint>
        <hint type="destinationlabel">
          <x>203</x>
          <y>138</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>stackedWidget1</sender>
      <signal>destroyed()</signal>
      <receiver>stackedWidget1</receiver>
      <slot>close()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>203</x>
          <y>138</y>
        </hint>
        <hint type="destinationlabel">
          <x>203</x>
          <y>138</y>
        </hint>
      </hints>
    </connection>
    <connection>
      <sender>dialog1</sender>
      <signal>destroyed()</signal>
      <receiver>dialog1</receiver>
      <slot>close()</slot>
      <hints>
        <hint type="sourcelabel">
          <x>203</x>
          <y>138</y>
        </hint>
        <hint type="destinationlabel">
          <x>203</x>
          <y>138</y>
        </hint>
      </hints>
    </connection>
  </connections>
</ui>